openslides/core/static/js/core.js

```diff
@@ -153,25 +143,39 @@ angular.module('OpenSlidesApp.core', [])
     return Autoupdate;
 })
 
-.factory('i18n', function($http) {
-    // TODO: there is a bug(?) in jed. I had to call val_idx++; in line 285
-    // TODO: make the language variable and changeable at runtime
-    var i18n = new Jed({
-        'domain': 'de',
-        'locale_data': {'de': {"": {}}},
-    });  // TODO: use promise here
-    $http.get('/static/i18n/de.json')
-        .success(function(data) {
-            // TODO: check data.
-            i18n.options.locale_data['de'] = data;
-        });
-    return i18n;
-})
-
 .factory('Config', function(DS) {
     return DS.defineResource({
         name: 'config/config',
         idAttribute: 'key',
         endpoint: '/rest/config/config/'
     });
+})
+
+.controller("LanguageCtrl", function ($scope, gettextCatalog) {
+    // controller to switch app language
+    // TODO: detect browser language for default language
+    gettextCatalog.setCurrentLanguage('en');
+    //TODO: for debug only! (helps to find untranslated strings by adding "[MISSING]:")
+    gettextCatalog.debug = true;
```

Comment:
We have to bundle this with the server debug settings. Can you create an issue for that?